packages/react-app/src/services/bounty.service.ts

```diff
@@ -12,14 +12,19 @@ export const getFilters = (query: NextApiQuery): FilterParams => {
 	 * Retrieve implemented filters from the query string params
 	 */
 	const filters = {} as FilterParams;
-	
+
 	if (typeof query.status === 'string') filters.status = query.status;
 	if (typeof query.search === 'string') filters.search = query.search;
 	if (typeof query.customerId === 'string') filters.customerId = query.customerId;
+	if (typeof query.userId === 'string') filters.userId = query.userId;
 	
 	if(query.lte) filters.$lte = Number(query.lte);
 	if(query.gte) filters.$gte = Number(query.gte);
 
+	if (query.createdByMe) filters.createdByMe = Boolean(JSON.parse(query.createdByMe as string));
+	if (query.claimedByMe) filters.claimedByMe = Boolean(JSON.parse(query.claimedByMe as string));
```

Comment:
I would probably have approached this slightly differently - adding `createdBy` and `claimedBy` and then passing the discordId of any user. 

Then the `created by me` functionality would just be a specific implementation of the above where the createdBy === current user id

Reply:
Changed it. See the latest commit